### PR TITLE
feat(NgClass) allow multiple classes per expression

### LINKED ISSUE #5205: InvalidCharacterError when adding multiple classes on single condition in ng-class
An InvalidCharacterError is thrown on constructs like the following (I believe due to the space character):

```
<i [ng-class]="{'fa-spinner fa-spin': isWorking}"></i>
```

Adding multiple classes on a single condition like this worked for me in 1.4. A workaround is to separate the two classes:

```
<i [ng-class]="{'fa-spinner': isWorking, 'fa-spin': isWorking}"></i>
```

### PR DESCRIPTION
**Pull request by ericmartinezr**
closes #5205

### CODE REVIEW

**Comment by ericmartinezr** [comment]
pls review

File: modules/angular2/test/common/directives/ng_class_spec.ts [hunk 1]
```diff
@@ -151,6 +151,25 @@ export function main() {
                  async.done();
                });
          }));
+
+
+      it('should allow multiple classes per expression',
```
Comment (pkozlowski-opensource) [inline modules/angular2/test/common/directives/ng_class_spec.ts]: We should have more tests here:
* make sure that both classes are removed when an expression flips to false
* what about spaces in class lists in lists / arrays?
* what happens if there is more that one space between class names?

Comment (ericmartinezr) [inline modules/angular2/test/common/directives/ng_class_spec.ts]: I'm on it. I'll back to you in case of any doubt

**Comment by ericmartinezr** [comment]
@pkozlowski-opensource updated with your points addressed, pls review again.

**Comment by pkozlowski-opensource** [comment]
LGTM

**Pull request closed without merging**